src/import-data/ImportData.ts

```diff
@@ -87,119 +103,133 @@ export const ImportData = () => {
     loadTemplateWorkspaces();
   });
 
```

Comment:
Changes to the various `importX` functions are mostly making them all take the same arguments: `importRequest` and `workspace` and having them actually do the import instead of returning a function that does the import.

Reviewing with whitespace changes ignored makes this clearer.